src/sdk/pynni/tests/test_model_speedup.py

```diff
@@ -104,6 +105,60 @@ def zero_bn_bias(model):
                 shape = module.running_mean.data.size()
                 module.running_mean = torch.zeros(shape).to(device)
 
+class L1ChannelMasker(WeightMasker):
```

Comment:
Shall we put `L1ChannelMasker` into the structure_pruning.py?  What is the reason for putting this class in the test module?

Reply:
This is just a simple channel pruning Masker to test channel pruning speedup.  Maybe move to structure_pruning.py when it is needed.